frontend/src/components/layouts/TwoColumnFullScreenLayout.tsx

```diff
@@ -72,7 +73,7 @@ export default function TwoColumnFullScreenLayout() {
       >
         <div className="flex-1 w-full h-full flex flex-col">
           <div className="flex flex-row justify-between items-center">
-            <h1 className="text-lg font-medium">Alby Hub</h1>
+            <AlbyHubLogo className="text-background" />
```

Comment:
Why text-background?

Reply:
This is the same color being used to highlight the Quote Author name in white, so I picked this

Reply:
Yeah, this one is needed because the background is the opposite of normal